README.md

```diff
@@ -1,6 +1,6 @@
-# item_party
+# Item Party
 
-The probably worst economic Minetest mod.
+The probably worst economic Minetest mod.    
```

Comment:
Not an improvement.

Reply:
Without this "improvement" (in markdown format):

        The probably worst economic Minetest mod. Various common nodes have an unusual behaviour:

With this "improvement" (in markdown format):
```
The probably worst economic Minetest mod. 
Various common nodes have an unusual behaviour:
```

Which one do you choose?

Reply:
til `   ` results in a line continuation rather than a full new line. Never mind then, no need to change.